src/components/Modal.tsx

```diff
@@ -57,7 +57,7 @@ export const Modal: FC<ModalProps> = ({
 										className="absolute top-0 left-[calc(49%)] -translate-x-1/2 -translate-y-[calc(43.2%)] w-[70%]"
 									/>
 									<div className="flex justify-end">
-										<button onClick={onClose}>
+										<button type="button" onClick={onClose}>
```

Comment:
The rationale behind this rule is this:
> The default type of a button is submit, which causes the submission of a form when placed inside a form element. This is likely not the behaviour that you want inside a React application.
>
> https://biomejs.dev/linter/rules/use-button-type/

Reply:
This is good practice to have all buttons explicitly set to `type="button"`. I know a lot of sites that don't do it and a button that is not submit triggers my password manager or something else which is annoying.